services/wallet/collectibles/manager.go

```diff
@@ -727,8 +727,16 @@ func (o *Manager) fillCommunityID(asset *thirdparty.FullCollectibleData) error {
 	return nil
 }
 
+func idsFromAssets(assets []*thirdparty.FullCollectibleData) []thirdparty.CollectibleUniqueID {
```

Comment:
bunch of similar functions in `services/wallet/collectibles/types.go`, please move this one there